### DISCUSSION TITLE
listing의 관리 방식 방향성에 대해

### DISCUSSION THREAD

**Discussion by josephjang** [rinthel/rust-lang-book-ko#253] [General]
이미 아시는 부분이리라 생각합니다만,

Rust book 한글 번역을 읽다가 오류가 있는 부분을 수정하려다보니,
한글 번역은 리스팅 (코드)들이 모두 본문에 포함된 채로 번역이 되었고,
업스트림은 리스팅들이 별도 파일로부터 rustdoc_include helper로 연결된다는 것을 깨닫게 되었습니다.

이에 대해서 한글 번역을 관리하시는 분들은 어떠한 방향으로 생각하고 계신지 알려주시면 감사하겠습니다.
당장은 본문 포함 방식을 유지하면서 관리하는 방향인 것인지,
chapter별로라도 업스트림처럼 rustdoc_include를 사용하도록 변환해나가는 방향인 것인지 궁금합니다.

만약 전자라면 현재 본문 상의 소스를 수정하는 방식으로, 후자라면 rustdoc_include를 이용해서 수정하는 방식으로 PR을 올리려고 생각하고 있습니다.

**Comment by rinthel** [comment]
@josephjang 늦게 답변 드려서 죄송합니다.

현재 빌드된 페이지 [rinthel.github.io/rust-lang-book-ko](https://rinthel.github.io/rust-lang-book-ko)에
올라온 번역은 2nd edition 이고, 이미 대부분의 번역 작업이 마쳐진 상태이기 때문에 여기의 코드는
되도록 그대로 유지할 예정입니다.
현재 지속적으로 번역중인 2018 edition의 경우, 별도의 listing 파일로 뽑는 수정을 마친 상태입니다.

만약 빌드된 페이지 내의 내용을 수정하고자 하시면 본문 상의 소스를 수정하는 형태로 PR을 보내주시면 감사하겠습니다 😄

**Reply by josephjang** [reply]
네, 2nd edition과 2018 edition의 차이였군요. 이해했습니다.
답변 감사드립니다.
